### ISSUE TITLE
Desenvolvimento do diagrama de classe

### ISSUE THREAD

**Issue by SebastianIzydorio** [ifpe-cti/servicon-web#4]
Temos que criar a classe `Cliente`, que será o quem contratará o serviço de um profissional, onde aí já temos mais duas classes, `Serviço` e `Profissional`, além disso precisamos do `Endereço`, que será o local do trabalho a ser feito. Além dessas classes, quais outras precisaremos? @lucasmendsc @MarcelaCCM

**Comment by lucasmendsc** [comment]
@SebastianIzydorio 

- Cliente/usuário :   
> que será o quem contratará o serviço de um profissional

- Profissional : Será aquele que vai se cadastrar com o intuito de prestar serviços,ser indicado e afins.

- Endereço : Endereço vai servir para profissional,cliente e serviço. Visto que,um cliente pode requisitar um serviço que não seja prestado necessariamente na sua casa.

- Serviço : Ligação entre todas as entidades,e descrições adicionais sobre o mesmo.

_Acredito que também seja pertinente a classe_  ***Pagamento***, _pois, poderia não somente automatizar e simplificar o pagamento como também poderíamos fazer com que a plataforma calcule algum tipo de monetização em cima do valor final._

**Comment by SebastianIzydorio** [comment]
Concordo com a implementação da classe `Pagamento`. Qual seriam os atributos dessa classe?

**Comment by lucasmendsc** [comment]
@SebastianIzydorio 
> Concordo com a implementação da classe Pagamento. Qual seriam os atributos dessa classe?

1. Cliente (que solicitou e recebeu o serviço)
2. Profissional(que prestou o serviço e vai receber o pagamento)
3. Status do pagamento(true pago,false em aberto)
4. Preço do serviço( valor total,que inclui a porcentagem condizente a monetização)
_5. Forma de pagamento_ a decidir como será feita.

**Comment by fabriciofx** [comment]
@lucasmendsc Você como arquiteto, deve definir quem vai cuidar dessa issue. Após definir, coloque um label para marcar o responsável pela issue.
